.github/workflows/ci.yml

```diff
@@ -216,13 +216,22 @@ jobs:
           cat ./.github/workflows/resources/poseidon-minimal.service | envsubst > ~/.config/systemd/user/poseidon.service
           systemctl --user daemon-reload
           systemctl --user start poseidon.socket
+        if: ${{ success() || failure() }}
       - name: Print Poseidon Failure logs
         if: failure()
-        run: journalctl -xen --no-pager
+        run: journalctl --user -xen --no-pager
       - name: Run e2e recovery tests
+        run: make e2e-test-recovery
+        if: ${{ success() || failure() }}
+      - name: Print Systemd Failure logs
         run: |
-          tail -f /var/log/syslog &
-          make e2e-test-recovery
+          /usr/bin/systemctl --user show poseidon.service -p NRestarts
+          journalctl --user -xe -u poseidon.service --no-pager
+        if: failure()
+      - name: Print Coverage files
```

Comment:
This name doesn't seem to fit the purpose completely. Otherwise, I wouldn't expect that a print statement stops my application 😅